ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,7 @@
+Thank you for submitting an issue!
+
+Please make sure you include the names of the affected language(s) in your issue title, so that we can get the correct maintainers to look at your issue. Here are some examples:
```

Comment:
besides language, maybe ask people to also mention other relevant things, like compiler version, OS, FlatBuffers version (1.4 vs master), etc.
So an example could be:
- Crash when accessing FlatBuffer [C++, gcc 4.8, OS X, master]